src/txValidators.ts

```diff
@@ -190,6 +204,9 @@ function *validateTxBody(txBody: TransactionBody): ValidatorReturnType {
     yield* validate(txBody.update === undefined, err(ValidationErrorReason.UNSUPPORTED_TX_UPDATE, 'transaction_body.update'))
     yield* validateOptional(txBody.validityIntervalStart, bind(validateUint64, 'transaction_body.validity_interval_start'))
     yield* validateOptional(txBody.mint, validateMint)
+    yield* validateOptional(txBody.collaterals, validateCollaterals)
+    yield* validateOptional(txBody.requiredSigners, validateRequiredSigners)
```

Comment:
The function `validateTxBody` should get a description about what we are actually validating. E.g. we don't care about hash length in `validateRequiredSigners`, it was probably checked before.